salt/modules/boto_vpc.py

```diff
@@ -487,6 +489,30 @@ def resource_exists(resource, name=None, resource_id=None, tags=None,
     except BotoServerError as e:
         return {'error': salt.utils.boto.get_error(e)}
 
+def wait_til_resource_exists(resource_type, resource_id, region=None, key=None, keyid=None, profile=None):
+    '''Wait for the resource to exist/be ready
```

Comment:
Can you change this to be on two separate lines? The `'''` and the `Wait for ...` should be split up.

Can you also add a `.. versionadded:: 2017.7.7` directive to these docs as well as a CLI example?